src/components/Popups/ReplyField/ItemList.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { SyntheticEvent } from 'react';
+import DatalistItem from 'box-ui-elements/es/components/datalist-item';
+import noop from 'lodash/noop';
+import { Collaborator } from '../../../@types';
+
+export type Props = {
+    activeItemIndex?: number;
+    filterString?: string;
+    itemRow?: JSX.Element;
+    items: Collaborator[];
+    onSelect: (index: number, event: React.SyntheticEvent) => void;
+    setActiveItemIndex?: (index: number) => void;
+};
+
+const DefaultItemRow = ({ item, ...rest }: Collaborator): JSX.Element | null =>
```

Comment:
This should be a separate component/file. We should try to maintain one component per file.